OpenEdXMobile/src/test/java/org/edx/mobile/test/Sha1UtilTest.java

```diff
@@ -0,0 +1,20 @@
+package org.edx.mobile.test;
+
+import org.edx.mobile.util.Sha1Util;
+import org.junit.Test;
+
+import static junit.framework.Assert.assertEquals;
+
+public class Sha1UtilTest extends BaseTestCase {
+    @Test
+    public void testSha1Encryption() {
```

Comment:
Would be nice (but not essential) to have some javadocs for this test suite that link back to the tested class/methods.